src/ocamlc_loc/ocamlc_loc.mli

```diff
@@ -0,0 +1,28 @@
+type loc =
+  { path : string
+  ; line : [ `Single of int | `Range of int * int ]
+  ; chars : (int * int) option
+  }
+
+type severity =
+  | Error
+  | Warning of
+      { code : int
+      ; name : string
+      }
+
+type message =
+  | Raw of string
+  | Structured of
+      { preview : string option
```

Comment:
What is `preview`?

Reply:
Preview is the excerpt taken from the location. E.g.
```
| ..
| ..
```

It's not  worth showing inside the editor because one can immediately jump to the location.

Reply:
Good idea. I renamed it